src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -466,18 +466,25 @@ const getStyles = cacheStyles((theme: Theme) => ({
 }))
 
 export const GuiPluginListScene = connect<StateProps, DispatchProps, OwnProps>(
-  (state, { route: { params } }) => ({
+  (state, props) => ({
     account: state.core.account,
     accountPlugins: state.account.referralCache.accountPlugins,
     accountReferral: state.account.accountReferral,
     coreDisklet: state.core.disklet,
     countryCode: state.ui.settings.countryCode,
     developerModeOn: state.ui.settings.developerModeOn,
-    disablePlugins: state.ui.exchangeInfo[params.direction].disablePlugins
+    disablePlugins: state.ui.exchangeInfo[props.direction].disablePlugins
   }),
   dispatch => ({
     updateCountryCode(countryCode: string) {
       dispatch(updateOneSetting({ countryCode }))
     }
   })
 )(withTheme(GuiPluginList))
+
+export function GuiPluginListBuyScene(props: Exclude<OwnProps, 'direction'>) {
+  return <GuiPluginListScene {...props} direction="buy" />
+}
+export function GuiPluginListSellScene(props: Exclude<OwnProps, 'direction'>) {
+  return <GuiPluginListScene {...props} direction="sell" />
+}
```

Comment:
This is not a great way to do it. Your component gets a `route` prop, and in there is a `name`. You can use that to figure out which direction is happening (route.name === 'pluginListSell'), and simply delete the `direction` prop altogether. This should also clear up the type errors.

Reply:
Even though we discussed that we can do this later... I ended up just doing it because upon further inspection, I missed some changes that were necessary anyway. Those changes are to change each `navigate()` call for each tab's default scene for the walletList. Since I'm touch each `navigate()` call in this PR anyway, I might as well implement the final piece